### DISCUSSION TITLE
Docker repo updated but latest Github release not?

### DISCUSSION THREAD

**Discussion by jaydee73** [acmesh-official/acme.sh#4910] [Q&A] [answered]
Hi folks,

I am using the docker version of acme.sh (https://hub.docker.com/r/neilpang/acme.sh/tags) and my Container Manager informed me some days ago that the repo was updated. In fact, the latest-tag was pushed 7 days ago.

But when I take a look at the github repo, the latest official release is 3.0.7 from over a month ago.

The dev branch got some updates about a week ago, but the docker repo isn't based on the dev branch, is it?

Has anyone some informations about this?

Regards,
Stefan

**Comment by neilpang** [comment, marked as answer]
The docker's  "latest" tag is not the  "latest release" on github.  It means the latest master code.
So, if the master branch is updated,  the  "latest" tag will be update.
So, Latest means latest code.
